spec/support/shared_contexts/aruba.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.shared_context 'uses aruba API' do
+  def random_string(options = {})
+    options[:prefix].to_s + SecureRandom.hex + options[:suffix].to_s
+  end
+
+  before(:each) do
+    klass = Class.new {
+      include Aruba::Api
+
+      def set_tag(tag_name, value)
+        self.instance_variable_set "@#{tag_name}", value
+      end
+
+      def announce_or_puts(*args)
+        p caller[0..2]
+      end
+    }
+    @aruba = klass.new
```

Comment:
I would prefer to use a `let` block to define `aruba` then use that (without the `@`) in the specs that depend on it.